### ISSUE TITLE
Running nightwatch in parallel - missing config

### ISSUE THREAD

**Issue by bitdepth** [Crunch-io/nightwatch-vrt#14]
When running the vrt plugin in a spawned parallel process the ``nightWatchClient.globals.test_settings.visual_regression_settings`` in ``generate-screenshot-file-path.js`` is empty. I have found a workaround for this by adding it manually to the nightwatch config like so.

```
globals: {
	"waitForConditionTimeout": 5000, // sometimes internet is slow so wait.
	// copied this config here because it seems to be missing when running nightwatch in parallel
	test_settings: {
		visual_regression_settings: {
			"generate_screenshot_path": generateScreenshotFilePath,
			"latest_screenshots_path": "test/vrt/latest",
			"latest_suffix": "",
			"baseline_screenshots_path": "test/vrt/baseline",
			"baseline_suffix": "",
			"diff_screenshots_path": "test/vrt/diff",
			"diff_suffix": "",
			"prompt": false,
			"always_save_diff_screenshot": false,
			"threshold": 0.0
		}
	}
}
```

I'm assuming these properties are not being copied into the global object for some reason when running in parallel but are copied into the global object when running in series. Please advise.

**Comment by enriquecastl** [comment]
@bitdepth We really appreciate your finding. We've created a PR #17  that changes where the  `visual_regression_settings` object should be defined. We'll still support reading this object from `globals.test_settings.visual_regression_settings` but the recommended path (in the documentation) is 
`globals.visual_regression_settings`.

That should fix the library when running the tests in parallel.

**Comment by enriquecastl** [comment]
closed in #17
